src/wymeditor/parser/xhtml-sax-listener.js

```diff
@@ -431,16 +430,6 @@ WYMeditor.XhtmlSaxListener.prototype.closeBlockTag = function(tag) {
         this._insideLI = false;
     }
 
```

Comment:
Removing this made no tests fail. It's not clear that we want to remove elements with just a br, anyway.